docs/content/documentation/getting-started/cli-usage.md

```diff
@@ -13,11 +13,17 @@ the command help by running `zola <cmd> --help`.
 Creates the directory structure used by Zola at the given directory.
 
 ```bash
-$ zola init my_site
+$ zola init [my_site]
```

Comment:
I think we can just have 2 lines rather than [].

```
$ zola init
$ zola init my_site
```

To match with the rest

Reply:
Ok i will add the tests and fix this now